### PR TITLE
Feature add frontend ci

### PR DESCRIPTION
**Pull request by sivchari**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:

I added lint check of frontend on GitHub Actions.

#### Which issue(s) this PR fixes:

Related #5

#### Special notes for your reviewer:

/label tide/merge-method-squash

### CODE REVIEW

**Comment by sanposhiho** [comment]
This should be after #7. I'll be back after that.

File: web/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "format": "prettier --write .",
     "lint:js": "eslint --ext \".js,.vue\" --ignore-path .gitignore .",
     "lint:css": "stylelint \"**/**.{css,scss,sass}\"",
-    "lint": "yarn lint:js lint:css",
+    "lint": "yarn run lint:js && yarn run lint:css",
```
Comment (sanposhiho) [inline web/package.json:13]: Could you please delete the change on package.json? This issue will be solved in #7 as well.

**Comment by sivchari** [comment]
@sanposhiho 

Thanks for comment.
I confirmed #7 is merged. So, I fixed and pushed.

File: .github/workflows/lint-frontend.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          node-version: 16.x
+      - uses: actions/cache@v2
+        with:
+          path: ${YARN_CACHE_FOLDER}
+          key: ${{ runner.os }}-yarn-cache-
+      - name: Install Yarn
+        run: npm install -g yarn
+      - name: Install
+        run: yarn install --frozen-lockfile
+      - name: Test
```
Comment (sanposhiho) [inline .github/workflows/lint-frontend.yml:35]:
```suggestion
      - name: Lint
```

File: .github/workflows/lint-frontend.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    defaults:
+      run:
+        working-directory: web
+    steps:
+      - name: Check Out
+        uses: actions/checkout@v2
+      - name: Setup Node.js v16
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
```
Comment (sanposhiho) [inline .github/workflows/lint-frontend.yml:26]: Does this really work? I cannot find the syntax `16.X` on the official readme of setup-node action.
https://github.com/marketplace/actions/setup-node-js-environment

```suggestion
          node-version: '16'
```

Comment (sivchari) [reply .github/workflows/lint-frontend.yml:26]: Yes, it works.
setup-node downloads from `https://nodejs.org/dist` . And v16.x is released from that.
If you are worry, I change `16.x` to `16`.

FYI

https://github.com/actions/setup-node/blob/04c56d2f954f1e4c69436aa54cfef261a018f458/src/installer.ts#L266-L298

https://nodejs.org/dist/

Comment (sanposhiho) [reply .github/workflows/lint-frontend.yml:26]: I see. But, yeah, please change it. Just to be safe in the future, let's go the way that the official suggests.

Comment (sivchari) [reply .github/workflows/lint-frontend.yml]: I fixed it.

**Review by sanposhiho — APPROVED** [review]
Thanks :)
/lgtm
/approve

**Pull request merged**